user_guide_src/source/changelogs/v4.1.2.rst

```diff
@@ -26,6 +26,7 @@ Changes:
 - ``Entity::mutateDate`` uses external cast handler ``DatetimeCast::get``.
 - In order for ``Config\**`` classes to get their respective properties' values from the ``.env``, it is now necessary to namespace the property with the name of the class. Previously, the property names are enough but now disallowed because it can get system environment variables, like ``PATH``.
 - The array helper ``_array_search_dot`` is now marked for ``@internal`` use. As this is used by ``dot_array_search``, users should not use ``_array_search_dot`` directly in their code.
+- ``CacheInterface::getMetaData()`` returns have been specified to be null or and array with at least the "expires" key (absolute epoch expiration); on misses the File, Memcached, and Wincache Handlers still return ``false`` which will become ``null` in a future release.
```

Comment:
UG check fails here

Reply:
I don't see it on mobile. "User Guide syntax check: successful in 1m 14s". Do you know what the problem is?

Reply:
Ohhh I see I missed a backtick